src/listen.js

```diff
@@ -135,11 +135,10 @@ module.exports = function(defaultFuncs, api, ctx) {
                 }
                 return;
               }
-
               break;
             case 'buddylist_overlay':
               // TODO: what happens when you're logged in as a page?
-              if(!ctx.globalOptions.updatePresence) {
+              if(!ctx.globalOptions.batchPresence) {
```

Comment:
Make here to if the updatePresence and batchPresence must be a true

Reply:
I'm having trouble keeping a persistent friend list that is accurate.  Every 2 minutes or so, Facebook will tell me a whole bunch of people who are actually online, are now offline.  I'm assuming in this case the 0 means their status hasn't changed.     This is code fix that?

Reply:
so how can i properly use batchPresence?